set.mm

```diff
@@ -465000,59 +465061,40 @@ its graph has a given second element (that is, function value).
-      WEUJZVULUXFVXDUXFVULUXGIUXFUXGUWAUWBYSYSXJAFVULUWCWAUWDUWEUWFUWJUWGUWHYTU
-      XEUXIUXKFHUWQUWRUXDWLUXFHTZUXIUXKYRUXEVXEUXHUXCAEVXEUXGUWSUXBUXFHUFVGVJOV
-      KYTUWIXRUWKUWLUWRUWSULUEZUWTUXCUAULUWMZIHVSVXFUWTVXGUAUWSUWNYBUWOUWP $.
+      XFUWQUEZUXQUXFVCTZAUXQUXGUNTZUYMUYNUXQUNUXGTUYOUXPUNUXGVDVEUNUXGVHVIUXFUW
+      QVFVGUYNABPLVJVKVLUYCUXTGUOZUFUGZTZCUPZGUWQUMZUXSULUEZUYHUYBUYSFGUWQUXFUY
+      PTZUYAUYRACVUBUXGUYQUXTUXFUYPUFVMVNMVOVPVUAUYTUYHVUAUYTUJZUYGFUWQVUCUYMUY
+      FAVUCUYMUYFUJZUJZAAFUXFUXGRVQSZVRSZVUFUXFUGZUXGUQVQSZUXFUGZVSZVTSZWOZVUEA
+      FVUGWOZVUMVUEUXTVUGUFUGZTZVUNVUAVUDVUPUYTVUAVUDUJZVUOVUFUXTVUQUYMVUFUNUXG
+      WASUEZVUOVUFTVUAUYMUYFWBZVUQUNWCUEZUXGWCUEZVUFWCUEZUNVUFWDUIZVUFUXGWDUIZV
+      URVUQWEVUQUXGVUQUYMUXGULUEZVUSIUXFWFWGZWHZVUQUXGRVVGRWCUEVUQWIWJWKZVUQUNU
+      XTVUFWDVUAUNUXTWDUIVUDVUARUXSRWLUEZVUAWMWJUXSWNUNRWDUIZVUAWPWJUXSWQXGWRZV
+      UQVUFUXTRURSZRVQSUXTVUQUXGVVLRVQVUQUYEUXTRUQUKSZURSUXGVVLVUQRUXSUQVUQWSZV
+      UQUXSVUAVUDWTXAZVUQXBZXCVUAUYMUYFXDVUQVVMRUXTURVVMRTVUQXEWJXFXHZXIVUQUXTR
+      VUQRUXSVVNVVOXJVVNXKXLZXMVUQVUFUXGVUQVUFVVHXNZVUQUXGVVFXOZVUQUNRXQUIVUFUX
+      GXQUIYHVUQRUXGVVIVUQWMWJZVVTXPXRUUAVURVUTVVAVVBXSVVCVVDUJUJVUFUNUXGUUBUUC
+      UUDIUXFVUFUUEXTVVRUUFYAVUEUYSVUPVUNUPGUWQVUGUYPVUGTZUYRVUPCVUNVWBUYQVUOUX
+      TUYPVUGUFVMVNCAFUYPWOZVWBVUNACFUYPGUUJMYBZAFUYPVUGUUGZUUHVOVUAUYTVUDUUIUY
+      MVUGUWQUEZVUCUYFIUXFVUFUUKUULZUUMYCVUEVWFVUHIUEZVUJIUEZVUNVUMUPZVWGVUAVUD
+      VWHUYTVUQUYMVUFUNUXGUUNSZUEZVWHVUSVUQVUFULUEZVVEVUFUQURSZUXGWDUIVWLVUQRUL
+      UEZVVERUXGWDUIZVWMVWOVUQUUTWJVVFVUQRVVLUXGWDVUQUNRURSRVVLWDVUQRVVNUUOVUQU
+      NUXTRVUQYIVUQVUFUXTWLVVRVVSUUPVWAVVKUUQUURVVQXMZVWOVVEUJVWPVWMRUXGUUSYDUV
+      AVVFVUQVWNVUIUXGWDVUQUXGRUQVQSZVQSVWNVUIVUQUXGRUQVUQUXGVVTUVBZVVNVVPUVCVU
+      QVWRUQUXGVQVWRUQTVUQUVDWJXFUVEVUQUXGVVTUVFYEVUFUXGYFYGVUFIUXFYJXTYAVUAVUD
+      VWIUYTVUQUYMVUIVWKUEZVWIVUSVUQVUIULUEZVVEVUIUQURSZUXGWDUIVWTVUQVVEVWPVXAV
+      VFVWQUXGUVGXTVVFVUQVXBUXGUXGWDVUQUXGUQVWSVVPUVHVUQUXGVVTYKYEVUIUXGYFYGVUI
+      IUXFYJXTYAVWCAFUYPJUOZKUOZVSZVTSZWOZUPVUNAFVUGVXEVTSZWOZUPVUNAFVUGVUHVXDV
+      SZVTSZWOZUPVWJGJKVUGVUHVUJUWQIIVWBVWCVUNVXGVXIVWEVWBAFVXFVXHUYPVUGVXEVTUV
+      JYLVOVXCVUHTZVXIVXLVUNVXMAFVXHVXKVXMVXEVXJVUGVTVXMVXCVXDVUHVXDVXMYMVXMVXD
+      YNYSXFYLYOVXDVUJTZVXLVUMVUNVXNAFVXKVULVXNVXJVUKVUGVTVXNVUHVXDVUHVUJVXNVUH
+      YNVXNYMYSXFYLYOUYPUWQUEVXCIUEVXDIUEXSCDVWCVXGQVWDADFVXFUYPVXEVTUVKNYBUVIU
+      VLYGYCVUEUXFVULTZAVUMYTVUEUYMVWPVXOVUCUYMUYFWBVUERUYEUXGWDVUERVVMUYEWDXEV
+      UEUQUYDRVUEUVMZVUEUXSUQVUEUXSVUAUYTVUDUVNZXOZVXPUVOVVIVUEWMWJVVJVUEWPWJVU
+      EUQUNUQURSUYDWDUVTVUEUNUQUXSUQVUEYIVXPVXRVXPVUEUXSVXQUVPVUEUQVXPYKUVQYPUV
+      RYPVUCUYMUYFXDUVSUYMVWPUJZVULUXFVXSUXFVULUXGIUXFUXGUWAUWBYQYQXTAFVULUWCWG
+      UWDUWEUWFUWJUWGUWHYRUXEUXIUXKFHUWQUWRUXDWTUXFHTZUXIUXKYTUXEVXTUXHUXCAEVXT
+      UXGUWSUXBUXFHUFVMVNOVOYRUWIYCUWKUWLUWRUWSULUEZUWTUXCUAULUWMZIHWFVYAUWTVYB
+      UAUWSUWNYDUWOUWP $.
```

Comment:
This doesn't drop ax-13. I believe what happened here is consequence of `SAVE PROOF /COMPRESSED`, which I always perform during the minimization activity. Most likely the original proof hasn't been recompressed, so that's why this quite huge shortening appears.

Reply:
Nice! I try to always minimize, which normally saves the proof in a compressed form. Maybe it was an omission.